src/VPolygon.jl

```diff
@@ -585,8 +585,8 @@ function minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
     mP = length(vlistP)
     mQ = length(vlistQ)
     i = 1
-    k = 1
-    j = 1
+    k = findfirst(==(σ(N[1, 0], P)), vlistP)
+    j = findfirst(==(σ(N[1, 0], Q)), vlistQ)
```

Comment:
Very neat :+1:
I was a bit surprised that `σ(N[1, 0], Q)` seems to be computed only once. Are we sure this is the case?